src/elements/content-sidebar/Sidebar.js

```diff
@@ -122,6 +128,44 @@ class Sidebar extends React.Component<Props, State> {
         }
     }
 
+    getIsOpen = () => {
+        const { isDefaultOpen } = this.props;
+        return this.isForcedSet() ? this.isForcedOpen() : !!isDefaultOpen;
+    };
+
+    addAnnotation() {
```

Comment:
It seems kind of unfortunate that the orchestration component is becoming more aware of the internals of its children. Do we have an end state for what this should look like?

Reply:
Yeah agreed. I think ideally we would have something like redux to manage the API state and have some HOC consuming the AnnotatorContext and updating the state data. I'll work on moving this to a HOC as a step in that direction, something like `WithSidebarAnnotations`